app/views/questions/index.html.erb

```diff
@@ -21,3 +21,4 @@
        <% end %>
      </tbody>
    </table>
+   <div class="center"><%= paginate @questions %></div>
```

Comment:
Seemed like after I tried other pagination strategies that I had dependencies from other paths that were somehow not allowing Kaminari to initialize.